test/tests/jobs/CreateJobTest.cpp

```diff
@@ -296,13 +296,17 @@ struct CreateJobTest : tpunit::TestFixture {
 
         // Get the job
         command.clear();
-        command.methodLine = "GetJob";
+        command.methodLine = "GetJobs";
```

Comment:
is there another test that tests GetJob? If not can you create a separate test for GetJobs?

Reply:
There's a whole `GetJob` test suite.